src/services/ssl.ts

```diff
@@ -2,19 +2,19 @@ import request from '@/utils/request';
 import { transformFetchListData, transformFetchItemData } from '@/transforms/global';
 import { SSL } from '@/models/ssl';
 
-export const fetchList = () => request('/api/ssl').then(data => transformFetchListData<SSL>(data));
```

Comment:
the umi proxy will take care this part.

Reply:
But in fact, the compiled dashboard still request `/api/ssl`, and apisix can't respond correctly.
This may be a configuration error?

Reply:
Oh yes, you are right 👍 

https://pro.ant.design/docs/deploy-cn

Reply:
BTW, if you change this line, we should update the proxy config too, so it could work in development environment.

Reply:
I still need to study the different configurations in the development and the production.

Reply:
https://pro.ant.design/docs/mock-api-cn

Reply:
Please keep the `api` prefix, then I will merge it soon :)

Here is an Nginx conf for reference: https://pro.ant.design/docs/deploy-cn